src/libraries/Common/src/System/Security/Cryptography/SlhDsa.cs

```diff
@@ -455,6 +528,8 @@ public byte[] ExportEncryptedPkcs8PrivateKey(ReadOnlySpan<byte> passwordBytes, P
         ///   <para>The private key is not exportable.</para>
         ///   <para>-or-</para>
         ///   <para>An error occurred while exporting the key.</para>
+        ///   <para>-or-</para>
+        ///   <para><paramref name="pbeParameters"/> does not represent a valid password-based encryption algorithm.</para>
```

Comment:
This reason does not seem to have been consistently applied.  e.g. ExportEncryptedPkcs8PrivateKeyPem(ROS-byte, pbeParameters)